### PR TITLE
To add ppc64le architecture tags

### PR DESCRIPTION
**Pull request by aishwaryabk**
This is an initial PR for adding `ppc64le` tags to tests that are tested on Power.

cc: @pravin-dsilva @sajauddin

### CODE REVIEW

File: features/admin/pod.feature [hunk 1]
```diff
@@ -10,6 +10,7 @@ Feature: pod related features
   @network-ovnkubernetes @network-openshiftsdn
   @proxy @noproxy
   @heterogeneous @arm64 @amd64
+  @ppc64le
```
Comment (aleskandro) [inline features/admin/pod.feature:13]: Hi, is it possible to add them in the same row as the other architectures?

**Comment by aleskandro** [comment]
There's an empty file (`=`) in the root of your branch: https://github.com/openshift/verification-tests/blob/52cdb04e9bb08e4689f105bae047e21dcdb11113/%3D

File: features/admin/quota.feature [hunk 2]
```diff
@@ -177,6 +182,7 @@ Feature: Quota related scenarios
   # @case_id OCP-11566
   @admin
   @inactive
+  @ppc64le
```
Comment (aleskandro) [inline features/admin/quota.feature:185]: for inactive cases, we never verified they work on other architectures than amd64, and we usually skip these in the CI. Are you sure to add them?

Comment (pruan-rht) [reply features/admin/quota.feature:185]: if it's inactive it's not going to be run...so there's probably no need to add it.

**Comment by pruan-rht** [comment]
@aishwaryabk  need to rebase

File: features/admin/pod.feature (hunk 1 above)
Comment (aishwaryabk) [reply features/admin/pod.feature]: @aleskandro Thanks for reviewing. Yes, I've added them in the same line as other architectures.

File: features/admin/quota.feature (hunk 2 above)
Comment (aishwaryabk) [reply features/admin/quota.feature]: I've added the ppc64le tag only where there are other architecture labels present

**Comment by aishwaryabk** [comment]
Thanks for reviewing. I have updated the PR

**Comment by liangxia** [comment]
We need to add such info in Polarion (e.g, in arch field for this case), then map that info into tags in the code.

**Pull request merged**